workers/unity/Packages/com.improbable.gdk.mobile/Android/Utility/AndroidDeviceInfo.cs

```diff
@@ -2,22 +2,22 @@
 
 namespace Improbable.Gdk.Mobile.Android
 {
+    public enum AndroidDeviceType
+    {
+        Unknown,
+        Physical,
+        Emulator
+    }
+
     public static class AndroidDeviceInfo
     {
         public const string AndroidStudioEmulatorDefaultCallbackIp = "10.0.2.2";
 
-        private enum DeviceType
-        {
-            Unknown,
-            Physical,
-            Emulator
-        }
-
-        private static DeviceType activeDeviceType = DeviceType.Unknown;
+        private static AndroidDeviceType activeDeviceType = AndroidDeviceType.Unknown;
 
-        public static bool IsEmulator()
+        public static AndroidDeviceType ActiveDeviceType()
```

Comment:
We should maybe wait for the other PR to go in first so we can address these API changes in the AndroidClientWorkerConnector here.

Reply:
agree with that

Reply:
I already have the commit with needed changes to other PR for compatibility. Will add it if this PR goes in first. Does that work @DanailPenev ?

Reply:
Would prefer to have it in one PR as it is an API change. having those two things in two PRs is a bit confusing. And it also shouldn't take that much more time to wait for the other PR to go in

Reply:
Sure, we can wait with this one